### ISSUE TITLE
Why the nested level limitation?

### ISSUE THREAD

**Issue by mbohgard** [avrora/key-del#10]
I got a deep object and need to delete a prop 5 levels deep. This is apparently "not supported" yet. How come? It's just recursion?

**Comment by afaq-sf** [comment]
Agreed. I also would use this module if it allowed configure this level depth.
